stdlib/filename.mli

```diff
@@ -140,4 +140,9 @@ val quote : string -> string
     Warning: under Windows, the output is only suitable for use
     with programs that follow the standard Windows quoting
     conventions.
- *)
+*)
+
+val extension : string -> string
+(** Get the extension of a file if there is one, includes the . when
```

Comment:
This sentence is not grammatically correct. Replace "includes" with "including".